tests/codex/sales/helpers.nim

```diff
@@ -0,0 +1,16 @@
+import std/algorithm
+
+import pkg/chronos
+import pkg/questionable
+import pkg/questionable/results
+
+import pkg/codex/sales/reservations
+
+proc allAvailabilities*(r: Reservations): Future[seq[Availability]] {.async.} =
+  var ret: seq[Availability] = @[]
+  without availabilities =? (await r.availabilities), err:
+    raiseAssert "failed to get availabilities, error: " & err.msg
+  for a in availabilities:
+    if availability =? (await a):
+      ret.add availability
+  return ret.reversed()
```

Comment:
Why reversed?

Reply:
It was an attempt to retrieve availability in the order added, to make the tests prettier :) But I don't think order is preserved in the repo, so likely this is not needed and can be removed.

Reply:
removed `.reversed` as it likely doesn't make a difference